### PR TITLE
Dashboard: Added QR code to dashboard.

### PR DESCRIPTION
**Pull request by sohailrajdev97**

### CODE REVIEW

File: utils/authentication.js [hunk 1]
```diff
@@ -10,16 +11,20 @@ var findOrCreate = function (accessToken, profile, provider, done) {
 			return done(err);
 		}
 		if (!user) {
+			qr.toDataURL(profile.emails[0].value, { errorCorrectionLevel: 'H' },  function(err, url){
 			user = new userService({
 				name: profile.displayName,
 				email: profile.emails[0].value,
-				token: accessToken
+				token: accessToken,
+				qrData: url
```
Comment (rogeredthat) [inline utils/authentication.js]: ### Configure QR generation for users who were configured before feature was added.
If possible refactor the `findOrCreate` method to compensate for data stored before feature addition. Users registered on my machine for example exist and will not enter the `if(!user)` code block. 
Devise a method which holds an array of all new fields and runs whatever functions corresponds to them.

Comment (sohailrajdev97) [inline utils/authentication.js]: Done !

**Comment by aero31aero** [comment]
image
Error on my part or a bug in the system?

**Comment by rogeredthat** [comment]
Bug in the system. Remember we encountered this when making the accounts page. The request object is not carrying the updated user. That is why this happens. If you login again, it'll show up
I'll make this an issue.

**Pull request merged**